tutorials/beginner/configuration.md

```diff
@@ -7,162 +7,174 @@ layout: tutorial-beginner
+Let's continue with the __Network Binding__, as this binding can be used in every setup from the start and will show you the basic configuration possibilities.
 With the network binding, you can define some of your network devices as things in order to use them in a rule for example, or just to see if they are online or offline and for how long.
-Scroll down or use the "Search" field to find the "Network Binding".
-Click on install, and the binding will be downloaded and installed automatically.
 
-![](images/picture_06.jpg)
+Scroll down or use the __search field__ to find the __Network Binding__.<br/>
+Click on __install__, and the binding will be downloaded and installed automatically.
+
+![](images/install_binding_1.png)
 
-![](images/picture_07.jpg)
+![](images/install_binding_2.png)
 
-Now that the binding is installed, switch back to the "Inbox".
+## Add things
+Now that the binding is installed, switch back to the ```Inbox```.
 If you click the "+" button, you now are able to choose the binding which you want to use for discovering new things.
 Because we only installed the network binding so far, it's the only binding to choose.
 
-![](images/picture_08.jpg)
+![](images/add_thing_1.png)
 
-As soon as you click on "Network Binding", the binding will start an automatic discovery of all (reachable) network devices in the same subnet as your openHAB installation resides via ICMP (ping).
+As soon as you click on "Network Binding", the binding will start an automatic discovery of all reachable network devices which are in the same subnet as the system running openHAB.
 
-![](images/picture_09.jpg)
+![](images/add_thing_2.png)
 
 The discovered "things" are now displayed in the inbox, and you can decide what to do next:
 
-- Add the device as a thing by clicking the check icon.
-- Ignore the device by clicking the crossed-out-eye icon.
-- Delete the device by clicking the trashcan icon.
+|    Icon     	|     Meaning       
+| ------------- | ------------- |
+| ![](images/check.png)		| add the device as a thing |
+| ![](images/ignore.png)	| Ignore the device. The device will not be shown in the inbox anymore. By clicking on	__SHOW IGNORED__ all ignored devices will be shown again	| 
+| ![](images/delete.png) | Delete the device. The device will appear on the inbox with the next scan  	|    
 
-The difference between ignoring and deleting discovered things is as follows:
 
-- If you ignore a thing, it disappears from the visible inbox.
-If you decide to add it at a later time, you can do so by scrolling to the bottom of the inbox and click the "SHOW IGNORED" button.
-- If you delete the thing, it won't be visible in the inbox any more.
-But if you start a new discovery process later, the thing will reappear.
+In this example, we only have one item with the IP ```192.168.139.130``` available. Click on the blue check icon to add the device.
```

Comment:
Please use one line per sentence.

Reply:
@Confectrician Hi i "just" did click on "Resolve conversation" hoping that i'm able to directly change the code in here. But apparently that didn't do anything?
Since i'm not yet that experienced with GitHub, could you tell me what i shall do? Make the changed and create a new PR or can I directly change the PR in here?

Thanks for helping ;-)